arbitrum-docs/launch-orbit-chain/orbit-sdk-introduction.md

```diff
@@ -0,0 +1,70 @@
+
+
+The Arbitrum Orbit SDK is a comprehensive toolkit designed for intermediate blockchain developers interested in building on the Arbitrum Orbit platform.
+
+This SDK facilitates the creation and management of your own Orbit chain using the advanced capabilities of [Viem](https://viem.sh), a modern alternative to traditional Ethereum libraries like _ethers.js_ and _web3.js_.
+
+## Arbitrum Orbit Chains
+
+### Capabilities of the SDK:
+
+- **Configuration and Deployment**: Simplifies the process of configuring and deploying Orbit chain's core contracts.
+- **Initialization and Management**: After deployment, the SDK helps you initialize your chain and adjust configuration parameters.
+- **Ongoing Support and Updates**: Future releases will bring new features, such as tools for creating custom dashboards and monitoring systems for your Orbit chain. Feel free to return to this page regularly for updates!
+
+
+Here are the steps you can take to create an Orbit chain with the Orbit SDK:
+
+### 1. Orbit Chain Deployment
+
+A primary function of the Orbit SDK is the deployment of a new Orbit chain. The initial step in setting up an Orbit chain involves determining the type that best fits your needs. We offer four distinct types of Orbit chains:
+
+1. **[Rollup Orbit Chains](/launch-orbit-chain/how-tos/orbit-sdk-deploying-rollup-chain.md)**: These Orbit chains offer Ethereum-grade security by batching, compressing, and posting data to the parent chain, akin to the <a data-quicklook-from="arbitrum-one">Arbitrum One</a> chain as a rollup chain. You can read more on the [orbit rollup deployment page](/launch-orbit-chain/how-tos/orbit-sdk-deploying-rollup-chain.md).
+
+2. **[AnyTrust Orbit Chains](/launch-orbit-chain/how-tos/orbit-sdk-deploying-anytrust-chain.md)**: AnyTrust Orbit chains are implementations of the Rollup protocol that lower costs by accepting a mild trust assumption. <a data-quicklook-from="arbitrum-anytrust-protocol">AnyTrust</a> chains rely on an external Data Availability Committee (DAC) to store data and provide it on-demand instead of using its <a data-quicklook-from="parent-chain">parent chain</a> as the Data Availability (DA) layer. Instead, they are stored by a <a data-quicklook-from="data-availability-committee-dac">Data Availability Committee (DAC)</a> to reduce transaction fees. The main differences between an AnyTrust chain and a rollup chain are:
+   
+   - **Data Availability**: AnyTrust chains utilize a unique approach, with members of a Data Availability Committee responsible for keeping transaction data, whereas rollup chains send the data back to the parent chain.
+
+   - **Security Guarantees**: AnyTrust chains offer different security levels that might be more suited for specific applications, like gaming and social media.
+
+   - **Fee Cost**: AnyTrust chains are significantly cheaper than Rollup chains because there's no need to send data to the parent chain, facilitating cheaper transaction fees.
```

Comment:
```suggestion
   - **Fee Cost**: AnyTrust chains are significantly cheaper than Rollup chains because there's no need to send the full transaction data to the parent chain, facilitating cheaper transaction fees.
```
Adding a small precision there, since data is still posted to the parent chain.

Reply:
This hasn't been addressed in the new PR